src/index.ts

```diff
@@ -405,14 +406,36 @@ function webpackDevServer(
       },
       ...options,
     },
-    // HMR setup with React and react-hot-loader
     plugins: [new webpack.HotModuleReplacementPlugin()],
+  };
+}
+
+// This is the legacy option for React projects. It requires you to use
+// hot wrapper from react-hot-loader at the root of an app.
+function reactHotLoader() {
+  if (process.env.STORYBOOK) {
+    return {};
+  }
```

Comment:
when are we in storybook env? Does that mean that we won't have any hot reloading when we are working with storybook open in dev mode?

Reply:
Storybook implements its own HMR. That's why we disabled in the past for the older option.

Reply:
But it's a good question. I think we should validate the setup against Storybook to be sure.

Reply:
okay then lets ship it and try it out, maybe in `datatap`